### DISCUSSION TITLE
Client.timeout could not execute request

### DISCUSSION THREAD

**Discussion by orangmuda** [projectdiscovery/nuclei#1633] [General]
## Issue 
an error occurred while processing the scanner using multiple subdomains on target.txt and causing an error, please follow the screenshots:
#
![Screenshot_20220223-014029_One UI Home](https://user-images.githubusercontent.com/91846073/155197721-f7d67812-0e16-4788-98f6-f6e46b2e6d9c.jpg)

**Comment by ehsandeep** [comment]
@byteofjoshua `Client.timeout` is warning that server does not responded to client request, mostly in case when the server is down or not accessible for any reason, this is expected.

**Comment by orangmuda** [comment]
@ehsandeep when i just scan one website with ``nuclei -u redacted`` it works and no error occurs, and when i use a url list like ``nuclei -l /path/list`` here the scanner has an error and the template is not dettect.

**Comment by orangmuda** [comment]
**Hi!** Any update for this issue ?
I still waited now.

**Reply by ehsandeep** [reply]
@orangmuda, I still don't see the issue part, as when you are running with a single target, it means the target is accessible, and when you are running against a list of hosts, there is the possibility that some of them will not respond so the warnings.

Let me know if you have any example list that I can use to reproduce the possible issue where the host is up, and you still see the errors?
